third_party/move/move-compiler-v2/src/experiments.rs

```diff
@@ -132,6 +132,11 @@ pub static EXPERIMENTS: Lazy<BTreeMap<String, Experiment>> = Lazy::new(|| {
             description: "Turns on or off a group of optimizations".to_string(),
             default: Given(true),
         },
+        Experiment {
+            name: Experiment::CFG_SIMPLIFICATION.to_string(),
+            description: "Whether to do the control flow graph simplification".to_string(),
+            default: Inherited(Experiment::OPTIMIZE.to_string()),
```

Comment:
Can we please set this to `Given(false)`, rebase and submit the PR? We do not want to accidentally land this with being on by default, without having run comparison testing on this (and other optimizations in the queue).

Reply:
done